packages/nuxt/src/core/runtime/nitro/renderer.ts

```diff
@@ -180,7 +180,11 @@ export default defineRenderHandler(async (event) => {
   }
 
   if (ssrError && event.node.req.socket.readyState !== 'readOnly' /* direct request */) {
-    throw createError('Cannot directly render error page!')
+    throw createError({
+      statusCode: 404,
+      message: process.dev ? '[nuxt] Cannot directly render error page.' : undefined,
+      statusMessage: 'Page Not Found: /__nuxt_error'
```

Comment:
I think we can remove it too (at least for prod) http clients have default message

Reply:
I'll follow up with another PR to do that, for clearer signage in changelog 👍